<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EurecarURV_Dlg</class>
 <widget class="QMainWindow" name="EurecarURV_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eurecar Unmanned Robot Vehicle Dialog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="view_camera">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>23</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_segnet">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>23</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_velodyne">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>284</y>
      <width>651</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbx_sensorbt">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Sensor Button</string>
    </property>
    <widget class="QPushButton" name="bt_camera">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_lrf">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>77</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LRF On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_ugv">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UGV Init</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_kinova">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>77</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Kinova Init</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_velodyne">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>124</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Velodyne On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_gps">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>124</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS On</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="msg_urv_status">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>4</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>4</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Segnet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>265</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Velodyne</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>311</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>URV System Message</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbx_script">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>12</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Scenario Script</string>
    </property>
    <widget class="QComboBox" name="cb_step_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>197</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>26</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mission Title</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_scenario_run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>227</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scenario Run</string>
     </property>
    </widget>
    <widget class="QListView" name="lsview_mission_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>178</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Step Title</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_mission_run">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>227</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_script_update">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>227</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Script Update</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grbx_patial_operate">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>25</y>
       <width>301</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Partial Operating</string>
     </property>
     <widget class="QGroupBox" name="grbx_scenario_partial_operating">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>21</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Scenario</string>
      </property>
      <widget class="QPushButton" name="bt_scenario_partial_run">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>30</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>28</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Scenario Step</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_scenario_step_f">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>48</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_scenario_step_s">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>48</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>~</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grbx_mission_partial_operating">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>105</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Mission</string>
      </property>
      <widget class="QPushButton" name="bt_scenario_partial_run_2">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>30</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>28</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mission Step</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_scenario_step_f_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>48</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_scenario_step_s_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>48</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>~</string>
       </property>
      </widget>
     </widget>
     <widget class="QRadioButton" name="rd_partial_scenario">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Scenario</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rd_partial_mission">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>65</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mission</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_urg_start">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>490</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>URG Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Mission_Script">
    <property name="title">
     <string>S&amp;cenario Script</string>
    </property>
    <addaction name="action_load_scenario_script"/>
    <addaction name="action_load_mission_script"/>
   </widget>
   <widget class="QMenu" name="menu_Sub_Dialog_Box">
    <property name="title">
     <string>&amp;Sub Dialog Box</string>
    </property>
    <addaction name="action_driving_dialog_box"/>
    <addaction name="action_manipulation_dialog_box"/>
   </widget>
   <addaction name="menu_Mission_Script"/>
   <addaction name="menu_Sub_Dialog_Box"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_load_scenario_script">
   <property name="text">
    <string>Load Scenario Script </string>
   </property>
  </action>
  <action name="action_driving_dialog_box">
   <property name="text">
    <string>&amp;Driving Dialog Box</string>
   </property>
  </action>
  <action name="action_manipulation_dialog_box">
   <property name="text">
    <string>M&amp;anipulation Dialog Box</string>
   </property>
  </action>
  <action name="action_load_mission_script">
   <property name="text">
    <string>Load Mission Script</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
